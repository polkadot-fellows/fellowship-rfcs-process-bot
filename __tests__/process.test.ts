import * as github from '@actions/github'
import { expect, test } from '@jest/globals'
import { config } from 'dotenv'

import processCmd from '../src/process'

config()

test('processCmd', async () => {
  const ctx = {
    owner: 'xlc',
    repo: 'RFCs',
    issue_number: 14
  }
  const octokit = github.getOctokit(process.env.GH_TOKEN!)
  expect(
    await processCmd(
      octokit,
      '/bot merge blockhash=0x39fbc57d047c71f553aa42824599a7686aea5c9aab4111f6b836d35d3d058162',
      ctx
    )
  ).toMatchSnapshot()
}, 100000)
