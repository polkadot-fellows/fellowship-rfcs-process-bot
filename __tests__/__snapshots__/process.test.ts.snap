// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bot 1`] = `"RFC_APPROVE(0014,ba8418fc46d251acfdd39604c5fe6ea3d69ed464ed4113d50e82ea1c712a44fc)"`;
